### PR TITLE
autotest.parallel_dd: add a parameter

### PR DESCRIPTION
**Pull request by YongxueHong**
Add the parameter blk_extra_params_stg for a given disk

Signed-off-by: Yongxue Hong <yhong@redhat.com>

ID: 1474001

### CODE REVIEW

**Comment by Zhengtong** [comment]
ack by: Zhengtong <zhengtli@redhat.com>

**Comment by hereischen** [comment]
Ack-by: Haotong Chen hachen@redhat.com

**Comment by XueqiangWei** [comment]
LGTM,
Acked-by xuwei@redhat.com

**Comment by aliang123** [comment]
Please update your commit, param "blk_extra_params_stg" is not only used for ppc64le and ppc64.

**Comment by aliang123** [comment]
Acked-by: Aihua Liang <aliang@redhat.com>

Test cmd:
   python ConfigTest.py --guestname=RHEL.7.4 --platform=x86_64 --verbose=yes --testcase=parallel_dd --driveformat=virtio_scsi,virtio_blk
Test Result:
 PASS 1-smp_4.1024m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.autotest.parallel_dd
PASS 2-smp_4.1024m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_scsi.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.autotest.parallel_dd

File: generic/tests/cfg/autotest_control.cfg [hunk 1]
```diff
@@ -109,6 +109,7 @@
         - signaltest:
             test_control_file = signaltest.control
         - parallel_dd:
+            blk_extra_params_stg = "serial=TARGET_DISK0"
```
Comment (luckyh) [inline generic/tests/cfg/autotest_control.cfg]: IDE drive seems not have the `serial` option for QEMU 0.12.

**Pull request closed without merging**